website/community/how-to-contribute/contribute-docs.md

```diff
@@ -21,5 +21,132 @@ sidebar_position: 1
 
 # Contribute Documentation
 
-TODO
+The [Fluss](https://alibaba.github.io/fluss-docs/) website presents Fluss and its community. It serves several purposes including:
 
+* Informing visitors about Fluss and its features as a streaming storage for real-time analytics
+* Encouraging visitors to download and use Fluss
+* Encouraging visitors to engage with the community
+* Providing comprehensive documentation for users and developers
+
+This guide provides information on how to contribute to the Fluss documentation.
+
+## Fork and Clone
+Fork the [Fluss repository](https://github.com/alibaba/fluss)
+    
+``` bash
+git clone https://github.com/<your-username>/fluss.git
+cd fluss
+```
+
+# Create a Branch
+    git checkout -b feature-branch-name
+
+# Install Dependencies
+   ```bash
+   cd website
+   npm install
+   ```
+
+# Directory Structure
+- The Fluss documentation is organized as follows:
+   ```
+     fluss/
+     └── website/
+     ├── docs/                    # Main documentation (current version)
+     │   ├── intro.md
+     │   ├── concepts/
+     │   ├── quickstart/
+     │   └── ...
+     ├── community/               # Community content (not versioned)
+     ├── versioned_docs/          # Previous versions of documentation
+     │   ├── version-0.6/
+     │   └── version-0.5/
+     ├── static/                  # Static assets
+     ├── build_versioned_docs.sh  # Script for building versioned docs
+     └── ...
+   ```
+# Validation (including versioned docs)
+- Edit or create Markdown files in the appropriate directories
+- Generate versioned docs (simulates what CI does)
+```bash
+./build_versioned_docs.sh
+```
+
+- Then build the complete site
+``` bash
+npm run build -- --no-minify
+```
+  
+# Preview Changes
+- Start the local development server to preview changes
+```bash
+   npm run start
+```
+- View your changes at [http://localhost:3000](http://localhost:3000)
+
+
+# Version Expressions
+   - Fluss documentation uses placeholder variables that are automatically replaced during the build process
+     - **FLUSS_VERSION**: Expands to the full version (e.g., "0.6.0")
+     - **FLUSS_VERSION_SHORT**: Expands to the short version (e.g., "0.6")
+
+# Versioned Documentation
+   - Only content in `website/docs/* `is versioned
+   - Each version corresponds to a branch (e.g., release-0.5, release-0.6)
```

Comment:
Add:
- Fixes for `website/docs/*` documentation related to old versions should be backported to their corresponding release-x.y branches.